src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipFileProcessor.java

```diff
@@ -22,28 +21,15 @@
 
     private static final Logger log = LoggerFactory.getLogger(ZipFileProcessor.class);
 
-    private final String publicKeyDerFilename;
-    private final String signatureAlg;
-
-    public ZipFileProcessor(
-        @Value("${storage.public_key_der_file}") String publicKeyDerFilename,
-        @Value("${storage.signature_algorithm}") String signatureAlg
-    ) {
-        this.publicKeyDerFilename = publicKeyDerFilename;
-        this.signatureAlg = signatureAlg;
-    }
-
     public ZipFileProcessingResult process(
         ZipInputStream zis,
         String containerName,
         String zipFileName
     ) throws IOException {
-        ZipStreamWithSignature signedZip = ZipStreamWithSignature.fromKeyfile(
-            zis, publicKeyDerFilename, zipFileName, containerName
+        ZipStreamWithSignature signedZip = new ZipStreamWithSignature(
```

Comment:
I'll rename this class (or delete, possible no longer makes sense to keep it) in separate PR.